portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -46,7 +46,7 @@
 /* gmac_SAM.[ch] is a combination of the gmac.[ch] for both SAM4E and SAME70. */
 #include "gmac_SAM.h"
 #include <sysclk.h>
-#include "phyhandling.h"
+#include "phyHandling.h"
```

Comment:
Was this intentional?

Reply:
Yes intentional. The header file has always been called `phyHandling.h`.